examples/migration_guide/bet_disk.py

```diff
@@ -0,0 +1,196 @@
+            "sectionalPolars": "sectional_polars",
+            "MachNumbers": "mach_numbers",
+            "liftCoeffs": "lift_coeffs",
+            "dragCoeffs": "drag_coeffs",
+            "tipGap": "tip_gap",
+            "initialBladeDirection": "initial_blade_direction",
+            "bladeLineChord": "blade_line_chord",
+        }
+
+        keys_to_remove = [
+            "axisOfRotation",
+            "centerOfRotation",
+            "radius",
+            "thickness",
+        ]
+
+        betdisks = []
+        cylinders = []
+        for number, data in enumerate(betdisk):
+            cylinder = {
+                "name": f"betcylinder{number+1}",
+                "axis": data["axisOfRotation"],
+                "center": data["centerOfRotation"] * length_unit,
+                "inner_radius": 0 * length_unit,
+                "outer_radius": data["radius"] * length_unit,
+                "height": data["thickness"] * length_unit,
+            }
+
+            updated_data = {
+                key_mapping.get(key, key): value
+                for key, value in data.items()
+                if key not in keys_to_remove
+            }
+
+            updated_data["twists"] = [
+                {"radius": twist["radius"] * length_unit, "twist": twist["twist"] * angle_unit}
+                for twist in updated_data["twists"]
+            ]
+
+            updated_data["chords"] = [
+                {"radius": chord["radius"] * length_unit, "chord": chord["chord"] * length_unit}
+                for chord in updated_data["chords"]
+            ]
+
+            updated_data["sectional_polars"] = [
+                {
+                    key_mapping.get(key, key): value
+                    for key, value in polars.items()
+                    if key not in keys_to_remove
+                }
+                for polars in updated_data["sectional_polars"]
+            ]
+
+            updated_data["alphas"] = updated_data["alphas"] * angle_unit
+            updated_data["omega"] = updated_data["omega"] * omega_unit
+            updated_data["chord_ref"] = updated_data["chord_ref"] * length_unit
+            updated_data["sectional_radiuses"] = updated_data["sectional_radiuses"] * length_unit
+
+            if "blade_line_chord" in updated_data:
+                updated_data["blade_line_chord"] = updated_data["blade_line_chord"] * length_unit
+
+            cylinders.append(cylinder)
+            betdisks.append(updated_data)
+
+    return betdisks, cylinders
+
+
+def save_to_file(bet_disk_list, cylinder_list, save):
+    """
+    Saving the information about BETDisks and Cylinders to their respective files.
+    """
+    if save is True:
+        for number, bet in enumerate(bet_disk_list):
+            with open(f"betdisk{number+1}.json", "w") as betdisk:
+                betdisk.write(json.dumps(bet.model_dump(), indent=4))
+            with open(f"cylinder{number+1}.json", "w") as cylinder:
```

Comment:
The output cylinder.json file doesn't seem to be used in the tutorial_BET_Disk.py file, can you also show how to use this file in the tutorial script?

Reply:
It is an extra file but one that if the user chooses to do so, can provide information on the cylinder entity used in creating the BET Disk

Reply:
OK, let's keep it then